source/reference/methods.txt

```diff
@@ -412,12 +412,17 @@ Cursor Methods
 
        .. arrayAccess
 
-   * - :method:`cursor.batchSize()`
+   * - ``cursor.batchSize()``
 
      - Controls the number of documents MongoDB will return to the
        client in a single network message.
 
-       .. clone
+       In the following example, the query returns results in batches of
```

Comment:
[nit] Consider tightening this up to "The following example query returns..."

Reply:
Done.